dev/tests/api-functional/phpunit.xml.dist

```diff
@@ -8,7 +8,7 @@
  */
 -->
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.1/phpunit.xsd"
```

Comment:
As can be seen in [`composer.json`](https://github.com/magento/magento2/blob/c2810e0d6014fd8aa787d24e5d46c6593437f00a/composer.json#L77), we are using `phpunit/phpunit:~6.2.0`, so the schema referenced here is actually not applicable anymore.